utilities/scan_logs.py

```diff
@@ -72,6 +72,26 @@ def scan_for_errors(log_file_path, error_strings):
                     raise LogScanningError('{} found!! Please review: {} '.format(word, log_file_path))
 
 
+def scan_for_pattern(logfile_path, pattern_list):
```

Comment:
Why not enhance existing scan_for_errors? The only new feature would be a list of errors in this case

Reply:
There is a difference other than that, that is why I used a another method. scan_for_errors will look for errors and raise exception if error is found where as scan_for_pattern look for pattern, it raise exception when pattern not found. It is reverse in raising exception for each one of them.